projects/storefrontapp-e2e-cypress/cypress/helpers/my-company/my-company.ts

```diff
@@ -0,0 +1,155 @@
+                row.text[i].forEach((text) => {
+                  switch (text) {
+                    case 'b2bcustomergroup':
+                      return cy.get('td').eq(i).contains('Customer');
+                    case 'b2bmanagergroup':
+                      return cy.get('td').eq(i).contains('Manager');
+                    case 'b2bapprovergroup':
+                      return cy.get('td').eq(i).contains('Approver');
+                    case 'b2badmingroup':
+                      return cy.get('td').eq(i).contains('Admin');
+                  }
+                });
+              } else {
+                cy.get('td').eq(i).contains(row.text[i]);
+              }
+            }
+          }
+        });
+      j++;
+    }
+  });
+}
+
+export function getListRowsFromBody(
+  body: any,
+  objectType: string,
+  rows: MyCompanyRowConfig[]
+) {
+  return body[objectType].map((data) => {
+    const table = { text: [] };
+    rows.map((row) => {
+      if (row.showInTable) {
+        if (Array.isArray(row.variableName)) {
+          row.variableName.forEach((variable) => {
+            // TODO: Improper imp.
+            if (variable === 'startDate') {
+              let foundText = getVariableFromName(variable, data);
+              if (row.useDatePipe) {
+                const foundDate = new Date(foundText);
+                foundText = getFormattedDate(foundDate);
+              }
+              table.text.push(foundText);
+            }
+          });
+        } else {
+          const foundText = getVariableFromName(row.variableName, data);
+          table.text.push(foundText);
+        }
+      }
+    });
+    return table;
+  });
+
+  function getVariableFromName(name: string, dataset: any) {
+    return name.split('.').reduce((p, c) => (p && p[c]) || null, dataset);
+  }
+
+  function getMonthPartFromDate(date: Date): string {
+    return date.toString().slice(4, 7);
+  }
+
+  function getFormattedDate(date: Date): string {
+    return `${getMonthPartFromDate(
+      date
+    )} ${date.getDate()}, ${date.getFullYear()}`;
+  }
+}
+
+export function waitForData(thenCommand, waitForCommand?): void {
+  waitForCommand;
+  cy.wait('@getData').then((xhr: any) => {
+    if (xhr.aborted) {
+      waitForData(thenCommand);
+    } else {
+      thenCommand(xhr?.response?.body);
+    }
+  });
+}
+
+export function verifyList(rows, rowConfig): void {
+  cy.get('cx-table').within(() => {
+    checkRowHeaders(rowConfig);
+    checkRows(rows);
+  });
+}
+
+/**
+ * Login as user with organization administration powers.
+ */
+export function loginAsMyCompanyAdmin(): void {
+  cy.requireLoggedIn({
+    user: 'linda.wolf@rustic-hw.com',
+    registrationData: {
+      firstName: 'Linda',
+      lastName: 'Wolf',
+      titleCode: '',
+      password: '12341234',
+      email: 'linda.wolf@rustic-hw.com',
+    },
+  });
+}
+
+export function ngSelect(formControlName: string, sortKey: string): void {
```

Comment:
This seems like a repetition from one of the files above. Also check if something similar already exists